web/src/lib/components/user-settings-page/appearance-settings.svelte

```diff
@@ -1,11 +1,66 @@
 <script lang="ts">
   import { fade } from 'svelte/transition';
-  import { colorTheme } from '../../stores/preferences.store';
+  import { colorTheme, locale } from '../../stores/preferences.store';
   import SettingSwitch from '../admin-page/settings/setting-switch.svelte';
+  import Locales from '$lib/assets/locales.json';
+  import SettingCombobox from '../admin-page/settings/setting-combobox.svelte';
+  import type { ComboBoxOption } from '../shared-components/combobox.svelte';
+  import { onMount } from 'svelte';
 
-  export const handleToggle = () => {
+  let time = new Date();
+
+  const fallbackLocaleCode = 'en-US';
+  const fallbackLocaleName = 'English (US)';
```

Comment:
I think it would make sense (for coherency) to put this into a fallbackLocale object as well.